src/main/java/ca/intelliware/ihtsdo/mlds/search/StandingStateFieldBridge.java

```diff
@@ -1,8 +1,8 @@
-package ca.intelliware.ihtsdo.mlds.search;
-
-public class StandingStateFieldBridge extends TranslatedEnumFieldBridge {
-	@Override
-	protected String getBundleKeyPrefix() {
```

Comment:
@jaykamal why has this one been removed?

Reply:
@dmcgihtsdo This code was used by FullTextQuery, now it has been deprecated in SB3 so, We have replaced with search session for search functionality implementation.